sky/utils/kubernetes/k8s_gpu_labeler_setup.yaml

```diff
@@ -58,8 +58,8 @@ data:
     from kubernetes import config
     
     canonical_gpu_names = [
-        'A100-80GB', 'A100', 'A10G', 'K80', 'M60', 'T4g', 'T4', 'V100', 'A10',
-        'P100', 'P40', 'P4', 'L4'
+        'A100-80GB', 'A100', 'A10G', 'H100', 'K80', 'M60', 'T4g', 'T4', 'V100', 
+        'A10', 'P100', 'P40', 'P4', 'L4', 'A6000'
```

Comment:
Is this A6000 the same as `RTXA6000` in other clouds? If the labelling is `A6000` by default in k8s, we should probably rename it for the other clouds as well in the future. : )

Reply:
Yes! `nvidia-smi` lists this as `Nvidia RTX 6000` - perhaps we can converge onto nvidia-smi's naming convention in the future?